tests_tf/test_utils.py

```diff
@@ -79,6 +81,23 @@ def test_other_classes_return_val(self):
         res_expected = [0, 1, 3, 4]
         self.assertTrue(res == res_expected)
 
+    def test_get_logits_over_interval(self):
+        import tensorflow as tf
+        model = cnn_model()
+        wrap = KerasModelWrapper(model)
+        x = tf.placeholder(dtype=tf.float32, shape=(1, 28, 28, 1))
+        adv_x = x
```

Comment:
Feeding `x` for `adv_x` should probably cause some divide by zero error or something because it means that the linear cross section plot doesn't have a direction to explore